api/clients/eigenda_client.go

```diff
@@ -84,27 +93,30 @@ func (m EigenDAClient) GetCodec() codecs.BlobCodec {
 // data, which is necessary for generating KZG proofs for data's correctness.
 // The function handles potential errors during blob retrieval, data length
 // checks, and decoding processes.
+// TODO: should we use a pointer receiver instead, to prevent unnecessary copying of the EigenDAClient struct?
```

Comment:
what do you mean by this?

Reply:
`func (m *EigenDAClient) PutBlob` instead of `func (m EigenDAClient) PutBlob`

Reply:
make sense

Reply:
Note: we decided to do all the breaking changes in a future PR.